src/analysis/retail/monk/windwalker/Guide.tsx

```diff
@@ -0,0 +1,93 @@
+import SPELLS from 'common/SPELLS';
+import { TALENTS_MONK } from 'common/TALENTS';
+import { SpellLink } from 'interface';
+import { GuideProps, Section, SubSection } from 'interface/guide';
+import { AplSectionData } from 'interface/guide/components/Apl';
+import { explanationAndDataSubsection } from 'interface/guide/components/ExplanationRow';
+import { RoundedPanel } from 'interface/guide/components/GuideDivs';
+import PreparationSection from 'interface/guide/components/Preparation/PreparationSection';
+import CombatLogParser from './CombatLogParser';
+import * as AplCheck from './modules/apl/AplCheck';
+
+export default function Guide({ modules, events, info }: GuideProps<typeof CombatLogParser>) {
+  return (
+    <>
+      <Section title="Core Spells and Buffs">
+        <MasteryGraph modules={modules} events={events} info={info} />
+        {modules.risingSunKick.guideSubsection}
+        {modules.fistsofFury.guideSubsection}
+        {modules.strikeoftheWindlord.guideSubsection}
+      </Section>
+      <Section title="Major cooldowns">
+        {modules.invokeXuen.guideSubsection}
+        {modules.serenity.guideSubsection}
+      </Section>
+      <Section title="Core Rotation">
+        The current iteration of Windwalker forces multiple different playstyles based on talents
+        chosen, and whether you are inside or outside of a{' '}
+        <SpellLink spell={TALENTS_MONK.SERENITY_TALENT} /> window.
+        <SubSection title="During Serenity">
```

Comment:
This assumes you're taking Serenity instead of SEF; do you want this to always show or be conditional?

Reply:
Gated based on SER talent taken, will re-evaluate how this section's phrasing etc can be cleaned up in the future to make the talent choices more clear etc